takwimu/templates/profile/profile_detail.html

```diff
@@ -55,16 +55,39 @@ <h1>Elections</h1>
         <div class="section-container">
           <section class="clearfix stat-row">
             <h2 class="header-for-columns">Votes Distribution </h2>
-
-            <div class="column-half" id="chart-pie-elections-valid_invalid_dist" data-stat-type="percentage" data-chart-title="Valid Votes Vs Invalid Votes" data-initial-sort="-value"></div>
-            <div class="column-half" id="chart-pie-elections-registered_accred_dist" data-stat-type="percentage" data-chart-title="Registered Voters Vs Total Votes Cast" data-initial-sort="-value"></div>
+            {% if elections.valid_invalid_dist.name != 'No Data Found' %}
+              <div class="column-half" id="chart-pie-elections-valid_invalid_dist" data-stat-type="percentage" data-chart-title="Valid Votes Vs Invalid Votes" data-initial-sort="-value"></div>
+              <div class="column-half" id="chart-pie-elections-registered_accred_dist" data-stat-type="percentage" data-chart-title="Registered Voters Vs Total Votes Cast" data-initial-sort="-value"></div>
+            {% else %}
+              {% include 'profile/no_data_available.html' %}
+            {% endif %}
           </section>
         </div>
         <div class="section-container">
           <section class="clearfix stat-row">
             <h2 class="header-for-columns">Total Votes Per Candidate </h2>
+            {% if elections.valid_invalid_dist.name != 'No Data Found' %}
+              <div class="column-full" id="chart-histogram-elections-candidate_dist" data-stat-type="scaled-percentage" data-chart-title="Percentage Votes Per Candidate" data-initial-sort="-value"></div>
+            {% else %}
+              {% include 'profile/no_data_available.html' %}
+            {% endif %}
+          </section>
+        </div>
+      </article>
 
-            <div class="column-full" id="chart-histogram-elections-candidate_dist" data-stat-type="scaled-percentage" data-chart-title="Percentage Votes Per Candidate" data-initial-sort="-value"></div>
+
+      <article class="clearfix">
+        <header class="section-contents">
+          <h1>Crops</h1>
+        </header>
+        <div class="section-container">
+          <section class="clearfix stat-row">
+            <h2 class="header-for-columns">Crop Production in Tonnes </h2>
+            {% if crops.crop_distribution.name == 'No Data Found' %}
+              {% include 'profile/no_data_available.html' %}
+            {% else %}
+              <div class="column-full" id="chart-pie-crops-crop_distribution" data-stat-type="percentage" data-chart-title="Crop Production" data-initial-sort="-value"></div>
```

Comment:
When you have a lot of crops, the chart may appear too long requiring vertical scrolls. No problem if there are no other options (just asking if other chart types like histogram didn't work as well).

![chart_too_long](https://user-images.githubusercontent.com/1779590/43582867-74a940e2-9666-11e8-8c4e-9d641ce59f9e.png)